docs/source/RegressionTestFarm.rst

```diff
@@ -126,9 +126,9 @@ Then I'd create each machine with the appropriate labels
     Added host:
         foo-amd.bazcorp.com
 
-    $ /usr/local/autotest/cli/autotest-rpc-client host create -t amd64 -b hostprovisioning foo-intel.bazcorp.com
+    $ /usr/local/autotest/cli/autotest-rpc-client host create -t intel64 -b hostprovisioning foo-intel.bazcorp.com
```

Comment:
I'm not sure, does the 'amd64' mean host arch is amd64, if so I think we need to keep this.

Thanks,
Xu